### ISSUE TITLE
Instructions for tracing do not include instructions for setting eval-pushlimit

### ISSUE THREAD

**Issue by Vaguery** [lspector/Clojush#255]
https://github.com/lspector/Clojush/wiki/Tracing-a-Push-program-with-Clojush doesn't actually explain how to change the number of steps, it just sort of says "you're on your own" at the end.

**Comment by Vaguery** [comment]
And, needless to say, I don't have any idea how.

**Comment by lspector** [comment]
I've added an explanation. Haven't had time to test it, but I think it's right.

**Comment by Vaguery** [comment]
I think  maybe it should be

```clojure
(reset-globals {:evalpush-limit 100000 :max-points 2000})
```

**Comment by lspector** [comment]
Yes I think you're right. Fixed and sorry for posting without testing.
